src/test/java/com/baidu/hugegraph/test/functional/AuthRestoreTest.java

```diff
@@ -303,19 +301,18 @@ public void testAuthRestoreByTypesWithAccessException() {
                 "--directory", filePath
         };
 
-        Assert.assertThrows(RuntimeException.class, () -> {
+        Assert.assertThrows(IllegalArgumentException.class, () -> {
             HugeGraphCommand.main(args);
         }, (e) -> {
             String msg = e.getMessage();
-            Assert.assertTrue(msg.startsWith("java.lang.IllegalArgumentException:"));
-            Assert.assertContains("if type contains 'access' then should " +
-                                  "contains 'group' and 'target'.", msg);
+            Assert.assertContains("if type contains 'access' then " +
+                                  "'group' and 'target' are required.", msg);
         });
     }
 
-    private void loadData(String restoreFilePath, String dataFilePath) {
-        List<String> list = FileUtil.read(FileUtil.configPath(dataFilePath));
-
-        FileUtil.writeText(DEFAULT_URL + restoreFilePath, list);
+    private void loadData(HugeType hugeType, String dataFilePath) {
+        List<String> list = FileUtil.read(FileUtil.configPath(DEFAULT_TEST_URL +
+                                                              dataFilePath));
+        FileUtil.writeText(DEFAULT_URL + hugeType.string(), list);
```

Comment:
define var for:
DEFAULT_TEST_URL + dataFilePath
DEFAULT_URL + hugeType.string()